### PR TITLE
fix golang lint errors in newer version golang-lint

### PR DESCRIPTION
**Pull request by hormes**
Signed-off-by: Fansong Zeng <fanster.z@gmail.com>

### Ⅰ. Describe what this PR does
Fix errors in newer version golang-lint. But this issue does not upgrade the golanglint version for there are also issues related to go version compatibility.

### CODE REVIEW

**Review by jasonliu747 — COMMENTED** [review]
`http://www.apache.org/licenses/LICENSE-2.0`

File: pkg/webhook/util/writer/atomic/atomic_writer.go [hunk 1]
```diff
@@ -84,33 +84,37 @@ const (
...
-//      symlink
-//  3.  The old version of the volume is walked to determine whether any
-//      portion of the payload was deleted and is still present on disk.
-//  4.  The data in the current timestamped directory is compared to the projected
-//      data to determine if an update is required.
-//  5.  A new timestamped dir is created
-//  6.  The payload is written to the new timestamped directory
-//  7.  Symlinks and directory for new user-visible files are created (if needed).
+//  1. The payload is validated; if the payload is invalid, the function returns
+//     2.  The current timestamped directory is detected by reading the data directory
```
Comment (FillZpp) [inline pkg/webhook/util/writer/atomic/atomic_writer.go:88]: Are the indentations of 2,5,7,8,9 correct?

**Review by eahydra — APPROVED** [review]
/lgtm

File: pkg/descheduler/framework/runtime/framework_test.go [hunk 2]
```diff
@@ -5,7 +5,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+	http://www.apache.org/licenses/LICENSE-2.0
```
Comment (jasonliu747) [inline pkg/descheduler/framework/runtime/framework_test.go:8]: 这个应该不需要更改

File: pkg/scheduler/plugins/coscheduling/controller/podgroup_test.go [hunk 3]
```diff
@@ -6,7 +6,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+	http://www.apache.org/licenses/LICENSE-2.0
```
Comment (jasonliu747) [inline pkg/scheduler/plugins/coscheduling/controller/podgroup_test.go:9]: 同上，这个应该不需要更改

**Pull request merged**